docs/pages/demo/datepicker/index.mdx

```diff
@@ -28,8 +28,11 @@ Will be rendered to modal dialog on mobile and textfield with popover on desktop
 
 #### Responsiveness
 
-Date/Time pickers experience is extremely different on mobile and desktop. Here is how components will look on different devices.
-The default `Datepicker` component is responsive, which means that `Mobile` or `Desktop` mode will be rendered according to device viewport.
+The date picker component is designed and optimized for the device it runs on.
+The "Mobile" version works best for touch devices and small screens.
+The "Desktop" version works best for mouse devices and large screens.
+
+By default, the `DatePicker` component uses a `@media (pointer: fine)` media query to determine which version to use.
```

Comment:
```suggestion
By default, the `DatePicker` component uses a `@media (pointer: fine)` media query to determine which version to use. This can be customized by `desktopModeMediaQuery` prop. 
```